velox/dwio/common/SelectiveColumnReader.h

```diff
@@ -299,7 +299,9 @@ class SelectiveColumnReader {
         values_->capacity());
 
     anyNulls_ = true;
-    bits::setNull(rawResultNulls_, numValues_);
+    if (!returnReaderNulls_) {
```

Comment:
You should not even call this function when `returnReaderNulls_` is true.  Can you check what DWRF is doing (should be some fast path) and do the same in ORC?

Reply:
@Yuhta Okay, let me see how DWRF is implemented.

Reply:
@Yuhta I tested the DWRF format in the same way and found the same problem. It should be introduced by this [MR]( https://github.com/facebookincubator/velox/pull/5443)

Reply:
The root cause of this problem is that RLEv2 does not implement the logic of fast path reading.

Reply:
Then we should implement the fast path